src/object/obj_class.c

```diff
@@ -26,8 +26,6 @@ static struct daos_obj_class  *oclass_resil2cl(struct daos_oclass_attr *ca);
 
 /**
  * Find the object class attributes for the provided @oid.
- * NB: Because ec.e_len can be overwritten by pool/container property,
- * please don't directly use ec.e_len.
  */
 struct daos_oclass_attr *
 daos_oclass_attr_find(daos_obj_id_t oid, bool *is_priv, uint32_t *nr_grps)
```

Comment:
Not this patch issue, but is_priv is a bit confusing for this API. If this API suppose to be exported, probably remove this parameter for export API, and add another one for internal usage.

Or we just remove is_priv from daos_obj_class, and use oc_id to check if the object class is reserved.

Reply:
I agreed, but need confirm if there is any external usage for this API @mchaarawi ?

Reply:
this is a private function and not used externally so it's fine to change it if needed